resources/profiles/BBL/process/0.06mm High Quality @BBL A1 0.2 nozzle.json

```diff
@@ -12,7 +12,7 @@
     "initial_layer_speed": "16",
     "outer_wall_acceleration": "2000",
     "outer_wall_speed": "60",
-    "sparse_infill_pattern": "crosshatch",
```

Comment:
What is the deal with constantly changing infill patterns? Are those profile files came from Bambu Studio?

Reply:
These are the profile files directly from Bambu Studio, so I guess Bambulab decided to change it.

Reply:
These changes are actually from Orca.
Can you revert these changes to keep it consistent?